scalableminds/webknossos
# Refactored dropdown item events

This PR moves the onClick handling of dropdown menu items out of the inner containers like div and span directly into the items. Thus making the code more clear and increasing the area that triggers the onClick event. These changes are also needed for issue  #3319 because for the fiddling with the layout of dropdown menus a consistency is needed.

### URL of deployed dev instance (used for testing):
- [https://refactordropdownitemevents.webknossos.xyz](https://refactordropdownitemevents.webknossos.xyz)

### Steps to test:
- check all dropdown items that where changed to trigger their onClick event correctly. 
- Also there should be no visual changes.


- check whether I found all dropdown menus and checked them.
I checked: 
    - tracing_actions_view
    - dataset_action_view (none)
    - view_modes_view (none)
    - comment_tab_view (none)
    - tree_hierarchy_view (none)
    - tree_tab_view
### comment
Please note that I removed `event.target.blur();` because the event that is passed to the called function no longer has the property target. And I am pretty sure that antd already handles the blur.
### Issues:
- fixes #3382 

------
